aviary/docs/getting_started/onboarding_level2.ipynb

```diff
@@ -519,7 +513,7 @@
    "id": "57986647",
    "metadata": {},
    "source": [
-    "For `FLOPS` and `GASP` missions, this method performs several calls to `set_val` on the trajectory for states and controls to seed the problem with reasonable initial guesses using `initial_guesses` within corresponding phases (e.g. `default_flops_phases.py` and `default_gasp_phases.py`). For `solved` missions, it performs similar tasks but for hard-coded state parameters. This is reasonable because a `solved` mission is actually a level 3 Aviary approach. We will cover it in [level 3 onboarding doc](onboarding_level3.ipynb) in the next page. Note that initial guesses for all phases are especially important for collocation methods.\n",
+    "For `height_energy` and `2dof` missions, this method performs several calls to `set_val` on the trajectory for states and controls to seed the problem with reasonable initial guesses using `initial_guesses` within corresponding phases (e.g. `default_flops_phases.py` and `default_gasp_phases.py`). For `solved` missions, it performs similar tasks but for hard-coded state parameters. This is reasonable because a `solved` mission is actually a level 3 Aviary approach. We will cover it in [level 3 onboarding doc](onboarding_level3.ipynb) in the next page. Note that initial guesses for all phases are especially important for collocation methods.\n",
```

Comment:
I had it as lowercase because that directly corresponds to the string value the enum uses. If I use uppercase here I would expect a user to try and use that in an input file, which won't work.

I will make an issue to investigate making AviaryValues non-case-sensitive for string to enum conversion, and in the meantime would like to keep this doc as-is.

Reply:
I believe that would break code elsewhere that uses mission method name to complete a path to a module. I can investigate and see how simple it would be to change without hardcoding too much logic there